src/lab4.py

```diff
@@ -21,9 +21,6 @@ def get_views(self):
             return self.__views
 
 
-    def __str__(self):
-            return f"{self.__artist}, {self.__song}, {self.__duration}, {self.__views}, {self.genre}, {self.awards}"
-
```

Comment:
Remove the duplicate __str__ method:

Delete lines 24-26


Keep only the initial __str__ method definition on lines 36-40.

Reply:
Hello! Red lines 24-26 are code that no longer exists, but was once present in the project